src/esn.calendar.libs/app/services/shells/calendar-shell.js

```diff
@@ -675,13 +675,6 @@ require('./valarm-shell.js');
       var needsModify = false;
       var attendees = this.vevent.getAllProperties('attendee');
 
-      if (this.organizer) {
-        var organizerMailto = calendarUtils.prependMailto(this.organizer.email);
-
-        attendees = attendees.filter(function(attendee) {
-          return organizerMailto && attendee.getFirstValue() !== organizerMailto;
-        });
-      }
```

Comment:
This method explicitly says "Change the partstat of all attendees (**except the organizer**) to a specific status. if emails is defined, change only attendees matching with emails.", so if you do this, we might not know what impact it will have for other features.